sections/browsers.include

```diff
@@ -59,11 +59,31 @@
   <a>creator browsing context</a>. Otherwise, the <a>browsing context</a> has no
   <a>creator browsing context</a>.
 
-  If a <a>browsing context</a> <var>A</var> has a <a>creator browsing context</a>, then the
-  <dfn>creator origin</dfn>, <dfn>creator URL</dfn>, and <dfn>creator base URL</dfn> are the
-  [=concept/origin=], [=document url|URL=], and <a lt='document base URL'>base URL</a>, respectively, of
-  the {{Document}} that was the <a>active document</a> of that <a>creator browsing context</a> at
-  the time <var>A</var> was created.
+  <p>If a <span>browsing context</span> <var>context</var> has a <span>creator browsing
+  context</span> <var>creator</var>, it also has the following properties. In what follows, let
```

Comment:
Also, please avoid line-wrapping content inside a link--makes it a bear to search-replace later...